Sources/Basics/Errors.swift

```diff
@@ -11,6 +11,7 @@
 public struct InternalError: Error {
     private let description: String
     public init(_ description: String) {
-        self.description = description
+        assertionFailure(description)
```

Comment:
I think we need to use `preconditionFailure` if we want this to show up in release builds as well, right? https://developer.apple.com/documentation/swift/1539616-assertionfailure